gui/include/motioncontrollers.h

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: LicenseRef-GPL-3.0-or-later-OpenSSL
+
+#ifndef CHIAKI_MOTIONCONTROLLERS_H
+#define CHIAKI_MOTIONCONTROLLERS_H
+
+// Taken from https://github.com/gabomdq/SDL_GameControllerDB
+// The assumption is that the PS3(sixaxis), DS3 and DS4 do have
+// a motion control component.
+
+QList<QString> chiaki_motion_controller_guid_list({
```

Comment:
Make this `static QSet<QString> ...` instead of QList, then you will need to use `.contains()` instead of `.indexOf()`. Also no need for an extra file, you can just drop it in controllermanager.cpp